Helper/PaymentMethods.php

```diff
@@ -326,46 +310,20 @@ protected function _getCurrentCountryCode($store, $country)
         return "";
     }
 
-    /**
-     * @param $store
-     * @return mixed|string
-     */
-    protected function _getCurrentLocaleCode($store)
```

Comment:
Why exclude the shopperLocale from the request to /paymentMethods?

Reply:
We can pass it as an optional parameter but in the API explorer the example excluded it so I assumed it's not necessary anymore